plugins/hls-class-plugin/src/Ide/Plugin/Class/CodeAction.hs

```diff
@@ -111,11 +111,17 @@ codeAction recorder state plId (CodeActionParams _ _ docId _ context) = pluginRe
                 $ use GetInstanceBindTypeSigs docPath
             implemented <- findImplementedMethods ast instancePosition
             logWith recorder Info (LogImplementedMethods cls implemented)
+            let groups =
+                    -- If the mininal def contains nothing, perhaps there are
+                    -- default implementations of methods, so the LSP offers to
+                    -- add stub implementations of all methods.
```

Comment:
As we have `cls`, we don't need to guess whether there are default method impls or not. We can use `classOpItems` to get the information.

Reply:
Thanks! I was not aware of `classOpItems` :+1: